src/org/zaproxy/zap/extension/pscanrulesAlpha/ContentSecurityPolicyMissingScanner.java

```diff
@@ -107,21 +113,39 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 			// Always report if the latest header isnt found,
 			// but only report if the older ones arent present at Low threshold 
 			Alert alert = new Alert(getPluginId(), Alert.RISK_LOW, Alert.CONFIDENCE_MEDIUM, //PluginID, Risk, Reliability

```

Comment:
Shouldn't the risk be changed to INFO if `cspROHeaderFound` is true? (per comment in the issue)